sklearn/gaussian_process/kernels.py

```diff
@@ -1787,3 +1787,166 @@ def is_stationary(self):
     def __repr__(self):
         return "{0}(gamma={1}, metric={2})".format(
             self.__class__.__name__, self.gamma, self.metric)
+
+
+class SelectDimensionKernel(Kernel):
+    """Wrapper for applying kernels on selected active dimensions of input data.
+
+    A thin wrapper to apply given kernel on only selected dimensions of
+    input data.
+
+
+    Parameters
+    ----------
+    kernel : a kernel object.
+        an instantiation of kernel object.
+
+    active_dim : a numpy integer indices to apply on input data.
+
+    """
+
+    def __init__(self, kernel, active_dim):
+        self.kernel = kernel
+        if type(active_dim) is not np.ndarray:
```

Comment:
is there a reason for not using np.asarray and supporting active_dim with type tuple?